### ISSUE TITLE
[Quest][Cosmetic] Doorway to the Abyss

### ISSUE THREAD

**Issue by neheri** [SunwellWoW/Sunwell-TBC-Bugtracker#1418]
Decription: Quest can be completed, so this is only cosmetic. 

How it works: You need to right click on a looted item and then left click on a portal.

How it should work: You should be able to right click on a portal directly. 

Source (you should point out proofs of your report, please give us some source): 
https://youtu.be/RnHGjly30m4?t=148

**Comment by Dootz** [comment]
weird thing, it works but only from one side :D

**Comment by neheri** [comment]
Quest works you can right click an item in your bag and then click the portal, so it's fine however it's less than ideal and that's why I report.

**Comment by Dootz** [comment]
yeah, i know. Just taking notes to save some time when i get back to it.
